bool printCapabilities
bool printOutputs
bool printControls
---
string capabilities
